scvi/module/_mrdeconv.py

```diff
@@ -217,14 +235,19 @@ def loss(
         px_rate = generative_outputs["px_rate"]
         px_o = generative_outputs["px_o"]
         gamma = generative_outputs["gamma"]
+        v = generative_outputs["v"]
 
         reconst_loss = -NegativeBinomial(px_rate, logits=px_o).log_prob(x).sum(-1)
 
         # eta prior likelihood
         mean = torch.zeros_like(self.eta)
         scale = torch.ones_like(self.eta)
-        glo_neg_log_likelihood_prior = -Normal(mean, scale).log_prob(self.eta).sum()
-        glo_neg_log_likelihood_prior += torch.var(self.beta)
+        glo_neg_log_likelihood_prior = (
+            -1e-4 * Normal(mean, scale).log_prob(self.eta).sum()
+        )
+        glo_neg_log_likelihood_prior += self.beta_prior_weighting * torch.var(self.beta)
+
+        v_sparsity_loss = self.l1_sparsity * torch.abs(v).mean(1)
```

Comment:
there is a mix of using positive and negative dimension indices throughout. I'd stick to one